scripts/testing/install_dependencies.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Anaconda has plenty of dependencies and because of that it's hard to set
+# environment to with Anaconda properly.
+# Run this script to install all the required dependencies for the autotools to be
+# able to call make commands. This will enable you to test anaconda more easily.
+#
+# To avoid messing your system feel free to use `toolbox` in fedora so all the
+# dependencies will be installed to the container instead of your system.
+# The toolbox package is in standard Fedora repository.
+#
+# For toolbox use case run:
+#
+#   $ toolbox create
+#   $ toolbox enter
+#   $ dnf install rpm-build
+#   $ sudo ./scripts/testing/install_dependencies.sh
+#
+#
+# For direct host installation run:
+#
+#   $ sudo dnf install rpm-build  # to get rpmspec binary
+#   $ sudo ./scripts/testing/install_dependencies.sh
+#
+
+
+# remove all problematic pieces from anaconda spec to be able to get dependencies
+cat ./anaconda.spec.in | sed 's/@PACKAGE_VERSION@/0/; s/@PACKAGE_RELEASE@/0/; s/%{__python3}/python3/' > /tmp/anaconda.spec
```

Comment:
Useless use of cat :-) `sed 'prog' file > destfile`

But more importantly, this is a script that likely runs on real systems, so no hardcoded file names in /tmp please. That can range anywhere from DoS to exploits. Please use `mktemp`.

What's problematic about `%{__python3}`?

Reply:
__python3 will break rpmspec, I added the same to the Dockerfile.